addons/website_event/tests/common.py

```diff
@@ -2,16 +2,43 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo.addons.event.tests.common import TestEventCommon
+from odoo.addons.mail.tests.common import mail_new_test_user
 
 
 class TestWebsiteEventCommon(TestEventCommon):
 
+    @classmethod
+    def setUpClass(cls):
+        super(TestWebsiteEventCommon, cls).setUpClass()
+
+        cls.company_main = cls.env.user.company_id
+        cls.user_event_web_manager = mail_new_test_user(
+            cls.env, login='user_event_web_manager',
+            name='Martin Sales Manager', email='crm_manager@test.example.com',
+            company_id=cls.company_main.id,
+            notification_type='inbox',
+            groups='event.group_event_manager,website.group_website_designer',
+        )
+
     def _get_menus(self):
         return set(['Introduction', 'Location', 'Register'])
 
-    def _assert_website_menus(self, event):
+    def _assert_website_menus(self, event, menu_entries=None):
         self.assertTrue(event.menu_id)
 
+        all_entries = self._get_menus()
+        if menu_entries is None:
+            menu_entries = all_entries
```

Comment:
Why not:
```
if menu_entries is None:
    menu_entries = self._get_menus()
```
?

(Or even better in method definition: menu_entries=set(['Introduction', 'Location', 'Register'])
but not sure if ok for stable)